drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -514,18 +514,27 @@ function DragAndDropEditBlock(runtime, element, params) {
                             'show_problem_header': $element.find('.show-problem-header').is(':checked'),
                             'item_background_color': $element.find('#item-background-color').val(),
                             'item_text_color': $element.find('#item-text-color').val(),
+                            'max_items_per_zone': $element.find("#max-items-per-zone").val(),
                             'data': _fn.data,
                         };
 
                         $('.xblock-editor-error-message', element).html();
                         $('.xblock-editor-error-message', element).css('display', 'none');
                         var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+
+                        $('.xblock-editor-error-message', element).css('display', 'none');
```

Comment:
@itsjeyd yes, nice catch - L521 and L522 totally filtered out from my attention; actually, we don't need this two lines added.